z_infra_provisioning/cloud_infra_center/ocp_upi/add-new-compute-node.yaml

```diff
@@ -150,7 +150,7 @@
     environment:
       KUBECONFIG: "./auth/kubeconfig"
     shell: |
-      openstack server show {{ os_compute_server_name }}-{{ new_worker_index }} -c addresses -f value | awk -F = '{print$2}'
+      openstack server list | grep {{ os_compute_server_name }}-{{ new_worker_index }} | grep -oP '(?<=)\d+(\.\d+){3}''
```

Comment:
please make sure the regex "(?<=)\d+(\.\d+){3}" fix all the case

Reply:
Yes, it will match all ipv4 address `X.X.X.X`.